roles/openshift_health_checker/library/etcdlogs.py

```diff
@@ -0,0 +1,95 @@
+# pylint: disable=missing-docstring
+
+"""Interface to journalctl"""
```

Comment:
> although in a more generic implementation you'd probably search for several things at the same time in a single unit's journal. but we can cross that bridge later.

@sosiouxme yeah, was thinking of implementing this, but was not sure how to handle "callbacks" for different "matchers". Ideally I would send an array of dictionaries, with each dictionary containing a regexp field, and a function to be called if that regex was matched, to the remote module.